api/v1alpha1/selfnoderemediation_types.go

```diff
@@ -32,10 +33,12 @@ type RemediationStrategyType string
 // SelfNodeRemediationSpec defines the desired state of SelfNodeRemediation
 type SelfNodeRemediationSpec struct {
 	//RemediationStrategy is the remediation method for unhealthy nodes
-	//currently "NodeDeletion" is deprecated and "ResourceDeletion" will always happen, regardless of which strategy is selected
-	//it will iterate over all pods and volume attachments related to the unhealthy node and delete them
+	//currently "NodeDeletion" is deprecated and it could be either "ResourceDeletion" or "OutOfServiceTaint"
```

Comment:
Please add fullstops and start new sentences uppercase, because new lines are stripped, which make this hard to read in the yaml.

Reply:
Thanks. I fix it.